ligo/errors.mligo

```diff
@@ -137,6 +140,12 @@
 (* Some issue with binary search in `observe` entrypoint. *)
 [@inline] let internal_observe_bin_search_failed = 322n
 
+(* Attempt to garbade collect a tick with non-zero liquidity net. *)
+[@inline] let internal_non_empty_position_gc_err = 323n
```

Comment:
It looks like this isn't used anywhere :thinking:

Reply:
This is an artifact, someone (me?) forgot to regenerate errors after updating `generate_error_codes.hs`.

Reply:
Oh I see, got it :ok_hand: